scopus-import/src/main/java/no/sikt/nva/scopus/conversion/model/CristinPersonContainer.java

```diff
@@ -0,0 +1,8 @@
+package no.sikt.nva.scopus.conversion.model;
+
+import java.net.URI;
+import no.sikt.nva.scopus.conversion.model.cristin.CristinPerson;
+
+public record CristinPersonContainer(AuthorIdentifiers authorIdentifiers, URI cristinId, CristinPerson cristinPerson) {
```

Comment:
This is stupid. At one point I need to lookup the cristin-id for a contributor, and the next step I need to either lookup the cristinURI or attempt to fetchCristinPersonByOrcid. So I needed something that links the scopus author identifiers to cristin-id.